src/common/node/fileSystem.ts

```diff
@@ -1,28 +1,33 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
-import * as fs from "fs";
+import * as nodeFs from "fs";
 import * as path from "path";
 import * as Q from "q";
 
 export class FileSystem {
+    private fs: typeof nodeFs;
```

Comment:
I only think we have these issues in a small number of cases (mainly variables for node modules [because we use classes that start with uppercase, so we don't have this issue for our own modules] and variables with getters/setters). So I'd rather rename the module to node<NodeModule> and do something else with the variables with getters/setters.